### DISCUSSION TITLE
What is required for SMB browsing support on Linux?

### DISCUSSION THREAD

**Discussion by pglizniewicz** [doublecmd/doublecmd#1633] [General]
What do I need, to get Windows Network browsing in Double Commander? I already have `gvfs-fuse` i `gvfs-smb`, but as I understand, these are needed for smb:// links, right? If I try to open Windows Network it will hang for a while and only show a single entry with "?".

Browsing the network works in Dolphin (Fedora 40, KDE Plasma 6)  and I've tested it with some tools, so I assume I'm missing some dependency for Double Commander:

```
 ~ nmblookup '*'
192.168.0.102 *<00>

 ~ nbtscan 192.168.0.102
Doing NBT name scan for addresses from 192.168.0.102

IP address       NetBIOS Name     Server    User             MAC address      
------------------------------------------------------------------------------
192.168.0.102    ARCHI            <server>  ARCHI            00:00:00:00:00:00
```

**Comment by alexx2000** [comment]
How do you open network?

**Reply by pglizniewicz** [reply]
"Open VFS list" button in the Disk Panel, double click "Windows Network".

**Reply by alexx2000** [reply]
It is a wrong way. See https://doublecmd.github.io/doc/en/faq.html#gvfs how to use Gvfs.
